src/test/testRunner.ts

```diff
@@ -130,7 +130,7 @@ class CoverageRunner {
             // This doesn"t impact tests, but is a concern if we had some integration
             // tests that relied on VSCode accessing our module since there could be
             // some shared global state that we lose.
-            decache(fullPath)
+            // decache(fullPath)
```

Comment:
This appears to fix all problems I have locally which avoids the need to set `NO_COVERAGE`. Then we can do this conditionally only on Windows (I guess), and CI can again get coverage reports for integ tests.

Reply:
What do you think about making this an option that is driven by the function caller? (Similar to the init test list)

Reply:
sure but then both callers have to check "platform == windows"

Reply:
That seems reasonable, would we want to leave it enabled for unit tests? I'm under the impression that integration tests are where the problems arise.

Reply:
Depends on how we define integ vs unit tests. Activating the extension is very useful for fast tests, not only integ tests...

Reply:
That sounds reasonable -- seems like decache is intended for use with pure unit tests, which isn't the main focus of this repo's tests.

Reply:
Looks like this change prevent coverage reports from being generated. So I guess it isn't Windows-only...?

```
==> Searching for coverage reports in:
    + .coverage
--> No coverage report found.
    Please visit http://docs.codecov.io/docs/supported-languages